### ISSUE TITLE
Round brackets in links

### ISSUE THREAD

**Issue by Avaray** [markedjs/marked#1377]
Lets say I have markdown code like this:
`[Hibernating](https://en.wikipedia.org/wiki/Hibernation_(computing))`

Text output will be `Hibernating)` and last bracket will be removed from link
`https://en.wikipedia.org/wiki/Hibernation_(computing`

I don't know how to explain it in other way. I belive you understand this issue.

**Comment by styfle** [comment]
This does not appear to be a problem with the latest version of marked.

```js
var marked = require('marked');
var md = '[Hibernating](https://en.wikipedia.org/wiki/Hibernation_(computing))';
var html = marked(md);
console.log(html);
// "<p><a href="https://en.wikipedia.org/wiki/Hibernation_(computing)">Hibernating</a></p>"
```

See this [demo](https://marked.js.org/demo/?text=%5BHibernating%5D(https%3A%2F%2Fen.wikipedia.org%2Fwiki%2FHibernation_(computing))%0A%0A&options=%7B%0A%20%22baseUrl%22%3A%20null%2C%0A%20%22breaks%22%3A%20false%2C%0A%20%22gfm%22%3A%20true%2C%0A%20%22headerIds%22%3A%20true%2C%0A%20%22headerPrefix%22%3A%20%22%22%2C%0A%20%22highlight%22%3A%20null%2C%0A%20%22langPrefix%22%3A%20%22language-%22%2C%0A%20%22mangle%22%3A%20true%2C%0A%20%22pedantic%22%3A%20false%2C%0A%20%22sanitize%22%3A%20false%2C%0A%20%22sanitizer%22%3A%20null%2C%0A%20%22silent%22%3A%20false%2C%0A%20%22smartLists%22%3A%20false%2C%0A%20%22smartypants%22%3A%20false%2C%0A%20%22tables%22%3A%20true%2C%0A%20%22xhtml%22%3A%20false%0A%7D) for a working example.

**Comment by UziTech** [comment]
This was fixed in PR #1135 and released in [v0.4.0](https://github.com/markedjs/marked/releases/tag/0.4.0)

**Comment by Avaray** [comment]
Sorry, I didn't realise I was using old version in my project.
